plugins/Goals/lang/en.json

```
@@ -68,30 +68,30 @@
         "Filename": "filename",
         "GoalConversion": "Goal conversion",
         "GoalConversions": "Goal conversions",
-        "GoalConversionsBy": "Goal %s conversions by type of visit",
-        "GoalIsTriggered": "Goal is triggered",
-        "GoalIsTriggeredWhen": "Goal is triggered when",
+        "GoalConversionsBy": "%s goal conversions by type of visit",
+        "GoalIsTriggered": "A goal is triggered",
+        "GoalIsTriggeredWhen": "A goal is triggered when",
         "GoalName": "Goal Name",
         "Goals": "Goals",
         "NGoals": "%s goals",
         "NRevenue": "%s revenue",
         "NItems": "%s items",
         "ManageGoals": "Manage Goals",
-        "GoalsOverview": "Goals Overview",
-        "GoalsOverviewDocumentation": "This is an overview of your goal conversions. Initially, the graph shows the sum of all conversions. %s Below the graph, you can see conversion reports for each of your goals. The sparklines can be enlarged by clicking on them.",
-        "GoalX": "Goal %s",
-        "HelpOneConversionPerVisit": "If a Page matching this Goal is refreshed or viewed more than once in a Visit, the Goal will only be tracked the first time the page was loaded during this visit.",
+        "GoalsOverview": "Overview of Goals",
+        "GoalsOverviewDocumentation": "An overview of your goal conversions. Initially, the graph shows the sum of all conversions. %s Below the graph, you can see conversion reports for each of your goals. The sparklines can be enlarged by clicking on them.",
+        "GoalX": "%s Goal",
+        "HelpOneConversionPerVisit": "If a Page matching this goal is refreshed or viewed more than once during a visit, the goal will only be tracked the first time the page was loaded during the visit.",
         "IsExactly": "is exactly %s",
-        "LearnMoreAboutGoalTrackingDocumentation": "Learn more about %1$s Tracking Goals in Matomo%2$s in the user documentation.",
-        "LeftInCart": "%s left in cart",
+        "LearnMoreAboutGoalTrackingDocumentation": "Learn more about %1$stracking goals%2$s in Matomo in the user documentation.",
+        "LeftInCart": "%s left in the cart",
```

Wouldn't say the `the` is needed here, as it doesn't describe a specific cart...
```suggestion
        "LeftInCart": "%s left in cart",
```

---

I thought it did describe a specific cart.
Seems a likely error to make in translation when it isn't specified?